examples/simple.js

```diff
@@ -84,53 +155,82 @@ d3.tsv("../data/demo.tsv", function(error, data) {
 
       // Extract the data as SVG text string
       var svg_xml = (new XMLSerializer).serializeToString(svg);
-});
 
+      var bars = d3.selectAll(".bar")
```

Comment:
This is because when we re-render a graph, it is actually a new one. We need to add the onclick event listener every time we draw a new graph.